scripts/auto-rebase/rebase-lvms.sh

```diff
@@ -181,6 +169,10 @@ download_lvms_operator_bundle_manifest(){
         local namespace="openshift-storage"
         extract_lvms_rbac_from_cluster_service_version "${PWD}" "${csv}" "${namespace}"
 
+        # Push the configMap to the kube-public namespace so that it is available to all users/apps
+        generate_version_config_map "${version}" "lvms-version" "kube-public"\
```

Comment:
Do we get one of these per ${ARCHS} now? Are the contents the same?

Reply:
The contents are identical but you use one staging environment per arch (not sure when or why this was introduced), so if i want to generate files I have to do this per arch.

Reply:
If you look at release-aarch64 / x86_64, the docker manifests are exactly the same because we build multi-arch containers, so Im not sure why one per arch is necessary

Reply:
Maybe that's a legacy from when we were using images that were not multi-arch. @pmtk would likely know.

Reply:
I recall something about osbuild not handling multiarch images correctly? @ggiguash is that correct?

Reply:
Im wondering how that fits together. LVMS has multiarch images since forever (4.12 I believe already had them), so im wondering how they wouldnt work with osbuild? Is that even relevant?

Reply:
Wait, I may have misread things.

Previously the file was created at `"${REPOROOT}/assets/components/lvms/topolvm-configmap_lvms-version.yaml"` and now it ends up there because you removed "ignore" in lvms-assets.yaml.

I don't think we should make that change - entries without `ignore` are the ones we obtain from lvms' CSV, this is not the case for the file - we generate it during rebase, in microshift repo.

Reply:
I discussed with @jakobmoellerdev what's going on here and now I know the story: version configmap was not generated correctly.

What Jakob proposed here is not strictly adhering to the flow we have in `rebase.sh` (that is: any file created during rebase, is created directly in assets, not in staging and then copied; like kustomizes per $ARCH for OLM).

However, I think proposed solution is more elegant - we don't need to use intermediate file where version gets dumped and then read during manifest update - it's generated during "download" which also extracts RBACs from CSV, then just copied with the power of `lvms_assets.yaml` and `handle_assets.py`. LGTM

Reply:
I agree. It's still reusing the existing flow, but in a less round-about way.